.gitattributes

```diff
@@ -8,3 +8,4 @@ docs/theme/js/theme.js linguist-vendored=false
 *_pb.d.ts linguist-generated
 *_pb.ts linguist-generated
 *_pb.grpc-*.ts linguist-generated
+*_pb.client.ts linguist-generated
```

Comment:
FYI, you'll have to update the bot as well, see https://github.com/gravitational/shared-workflows/pull/220.

Reply:
Just a reminder about updating the bot.

Reply:
Yeah, I will open a PR soon.

Reply:
https://github.com/gravitational/shared-workflows/pull/234